sqlmesh/core/audit/definition.py

```diff
@@ -282,3 +428,7 @@ def _maybe_parse_arg_pair(e: exp.Expression) -> t.Tuple[str, exp.Expression]:
     if isinstance(e, exp.EQ):
         return e.left.name, e.right
     raise_config_error(f"Invalid defaults expression: {e}", error_type=AuditConfigError)
+
+
+def is_audit(node: Node) -> bool:
```

Comment:
I think this is used in exactly one place. Do we really need a function for this?

Reply:
I did a grep and found it used 6 times.